crt_portal/cts_forms/migrations/0076_auto_20200616_1010.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.13 on 2020-06-16 14:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0075_add_default_response_templates'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
This missing migration bothers me, does this mean that the trends page doesn't work in production or will not work for this release?

Reply:
Since this isn't a managed model (`managed=False`), I would expect the trends page to function as it does on dev and staging. 

This migration only keeps Django's view of the models/migrations in-sync with the code itself, no changes on the database.